app/2024/code-of-conduct/page.jsx

```diff
@@ -254,16 +254,26 @@ export default function CodeOfConduct() {
           contact:
         </p>
         <ul className="list-disc list-inside md:list-inside">
-          <li>Hassan Kibirige, Chairperson - <span className="font-bold">has2k1@gmail.com</span></li>
-          <li>Kirabo Atuhurira, Chairperson - <span className="font-bold">kiraboismyname@gmail.com</span></li>
+          <li>
+            Hassan Kibirige, Chairperson -{" "}
+            <span className="font-bold hover:underline">
+              <a href="mailto:has2k1@gmail.com">has2k1@gmail.com</a>
```

Comment:
you could actually use `Link` component from `next/link` given that it implements the same native anchor link.

```js
import Link from 'next/link'

<Link className="font-bold hover:underline" href="mailto:has2k1@gmail.com">has2k1@gmail.com</Link>
```

Reply:
this should be the last change. 

thanks @Leon12-aaryson

Reply:
could I do the import on that same page?

Reply:
yes please, at the top level

Reply:
push done, you can now review it